src/index.html

```diff
@@ -7,9 +7,27 @@
     <title>Meals homepage</title>
 </head>
 <body>
-    <header><h1> Hello this is meals project</header>
-    <main>
+<header class="d-flex-lg justify-between-lg p-1-lg">
+    <div class="logo">
+        logo
+    </div>
+    <nav>
+        <ul class="d-flex-lg">
+            <li><a href="#" class="p-1-lg"></a>Home</li>
+            <li><a href="#" class="p-1-lg"></a>About</li>
+            <li><a href="#" class="p-1-lg"></a>Recipes</li>
+            <li><a href="#" class="p-1-lg"></a>Blog</li>
+            <li><a href="#" class="p-1-lg"></a>Contact</li>
+        </ul>
```

Comment:
Consider wrapping the link name inside the `<a>` tag. so that it can be clickable.